README.md

```diff
@@ -39,7 +39,7 @@ To run a Docker container with [Work Directory](https://github.com/jenkinsci/rem
 Optional environment variables:
 
 * `JENKINS_JAVA_BIN`: Path to Java executable to use instead of the default in PATH or obtained from JAVA_HOME
-* `JENKINS_JAVA_OPTS` : Java Options to use for the remoting process, otherwise obtained from JAVA_OPTS
+* `JENKINS_JAVA_OPTS` : Java Options to use for the remoting process, otherwise obtained from JAVA_OPTS, **Warning!** :exclamation: mandatory double quotes around this value for windows.
```

Comment:
This is not actually true unless the options don't meet the PowerShell command line parsing requirements. For example, `-XX:+PrintCommandLineFlags`. In PowerShell the `:` is processed as a command line split character, whereas if you just had something like `--show-version` as the command line option, you do NOT need the quotes around it. The quotes need to be around arguments that would be processed by PowerShell in a way that would split them or be processed incorrectly, thus if you wanted something with both --show-version and -XX:+PrintCommandLineFlags, you only need to quote the -XX:+PrintCommandLineFlags like this: `--show-version "-XX:+PrintCommandLineFlags"`

Reply:
Does it make sense to recommend quoting each element? I feel like that opening the door of "quote only when required" is a recipe for disaster for end users.

Reply:
Sure, that wouldn't hurt, though sometimes you have embedded quotes as well, so that needs to be taken into account in the doc, e.g., `-Dsomething.something="Some string with spaces"`, which would need either the opposite quote character embedded (`"-Dsomething.something='Some string with spaces'"`) or the quotes would need to be escaped (`"-Dsomething.something=`"Some string with spaces`""`)

Reply:
> Not sure how to escape backticks, there should only be one backtick in the last example.

Use double backticks to wrap the inline code:

````
``"-Dsomething.something=`"Some string with spaces`""``
````

renders like this:

``"-Dsomething.something=`"Some string with spaces`""``


---- 

Note that it is the same idea for code blocks: use quadruple-backticks instead of triple, if you need to show a code block in the code block:

`````
````
```powershell
echo $env:JAVA_HOME
```
````
`````

renders like this:

````
```powershell
echo $env:JAVA_HOME
```
````